src/store/modules/order/actions.ts

```diff
@@ -0,0 +1,38 @@
+import { OrderService } from "@/services/OrderService";
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import OrderState from './OrderState'
+import * as types from './mutation-types'
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+
+
+const actions: ActionTree<OrderState, RootState> = {
+
+  async findPurchaseOrders ({ commit, state }, payload) {
+
+    let resp;
+    try {
+      resp = await OrderService.fetchPurchaseOrders(payload)
+
+      if (resp.status === 200 && !hasError(resp) && resp.data.grouped) {
+        const orders = resp.data.grouped.orderId
+        
+        if (payload.json.params.start && payload.json.params.start > 0) orders.groups = state.purchaseOrders.list.concat(orders.groups);
+        commit(types.PURCHASE_ORDER_UPDATED, {
+          list: orders.groups,
+          total: orders.ngroups
+        })
+      } else {
+        //showing error whenever not getting Orders
+        showToast(translate("Orders not found"));
```

Comment:
I think we have not added it to en.json. Please ensure it for other text as well